projects/plugins/jetpack/_inc/client/components/jetpack-product-card/test/component.js

```diff
@@ -43,29 +43,32 @@ describe( 'Jetpack Product Card', () => {
 
 	it( 'price is shown', () => {
 		const { container } = render( <JetpackProductCard { ...mockAttributes } /> );
+		
 		const priceObject = getCurrencyObject( mockAttributes.price, mockAttributes.currencyCode );
 
-		expect( getNodeText( container.querySelector( '.jp-product-card__currency-symbol' ) ) ).to.be.equal( priceObject.symbol );
-		expect( getNodeText( container.querySelector( '.jp-product-card__price-integer' ) ) ).to.be.equal( priceObject.integer );
-		expect( getNodeText( container.querySelector( '.jp-product-card__price-fraction' ) ) ).to.be.equal( priceObject.fraction );
-		expect( screen.getAllByText( mockAttributes.billingDescription ) ).to.exist;
+		expect( container.getAllByText( priceObject.symbol ) ).to.exist;
```

Comment:
We can't target elements by classes with `ProductPrice` anymore, since classes are automatically generated.